### PR TITLE
FAQ Statistics - add sentence

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

**Comment by dsarkar** [comment]
EN

image

**Comment by dsarkar** [comment]
DE

image

File: src/data/faq.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
                         "active": true,
                         "textblock": [
                             "Due to the high dynamics of the pandemic, the Corona-Warn app displays important statistical key figures on the current occurrence of the infection starting with the 1.11 version of the app. The shown figures and graphs should guide users to better assess the current situation and to adjust their own behaviour.",
+                            "The number of new infections is published once a day by the RKI. ' Yesterday' means that the value considers all reported new infections from the previous day until 23:59. ", 
                             "The statistics displayed in the Corona Warn app get compiled from different sources and get formatted for the app about every 15 minutes. Besides the aggregated numbers from the previous day (e.g., reported new infections), the statistics also show the R-value, which is an estimate for the current day based on the  \"nowcasting\" approach.",
```
Comment (MikeMcC399) [inline src/data/faq.json:46]: In the previous line (44), change Corona-Warn app to Corona-Warn-App to align with other uses of the name of the app in the English FAQs.

In line 46 change Corona Warn app to Corona-Warn-App for the same reason as above.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I have suggested using the standard name for the app: Corona-Warn-App, which aligns also with the naming in the app stores. The name is the same in the English and the German app UIs.

**Review by GisoSchroederSAP — APPROVED** [review]
Fine for me.  Good enhancement, uncritical, but beneficial.

**Review by loki-cui — APPROVED** [review]

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json]: ok

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks.

**Pull request merged**